app/src/org/commcare/preferences/CommCarePreferences.java

```diff
@@ -420,6 +420,16 @@ public static double getGpsCaptureAccuracy() {
         }
     }
 
+    public static int getGpsAutoCaptureTimeout() {
+        SharedPreferences properties = CommCareApplication._().getCurrentApp().getAppPreferences();
+        try {
+            return Integer.parseInt(properties.getString(GPS_AUTO_CAPTURE_TIMEOUT,
```

Comment:
Do we want wrap this value in `TimeUnit.MINUTES.toMillis`, or have users input time in milliseconds?